Staffbase/trigger.io-tabs
# Ma 8080 upload support

The PR adds support for file input type in the Web View for android.

To Test:
- Open Frontend Repo
- Change "tabs" in client/config/trigger.io/config.json to "sb_tabs"
- Change Version of sb_tabs to 2.32.8
- Build a native android app: grunt run_android:staffbasetest --ac=test
- Install app on device.
- Open a link which contains file upload with "External Link" plugin in the app
- Check if file chooser works fine
- Verify that mime types are also respected for input field
